ui/src/kapacitor/containers/KapacitorRulesPage.tsx

```diff
@@ -1,17 +1,35 @@
-import React, {Component} from 'react'
-import PropTypes from 'prop-types'
+import React, {PureComponent, SFC} from 'react'
+
 import {connect} from 'react-redux'
 import {bindActionCreators} from 'redux'
 
-import {getActiveKapacitor} from 'shared/apis'
+import {getActiveKapacitor} from 'src/shared/apis'
```

Comment:
shared is a top level alias. same as below and above. are you finding that it's not working for some reason now? if it is working, i'd prefer to keep these as `shared` to cut down on unnecessary boilerplate, or otherwise i vote that we remove the various top level aliases since there's such inconsistent usage.

Reply:
I'm getting IDE errors, and haven't found a way to get rid of them, so keeping it for now. commented in code channel to get feedback.